storage/client/client.go

```diff
@@ -34,6 +37,39 @@ type StorageClient struct {
 	logger *log.Logger
 }
 
+// NumericToBigInt converts a pgtype.Numeric to a big.int using the private method found at
+// https://github.com/jackc/pgtype/blob/master/numeric.go#L398
+func (c *StorageClient) NumericToBigInt(ctx context.Context, n *pgtype.Numeric) (*big.Int, error) {
```

Comment:
If we change the contract of this func so that it returns `(0, err)` (instead of `(nil, err)`) when there's an error, then the return value can be `(big.Int, error)` (no pointer!) and most callsites simplify because you don't need an intermediate variable to write into a field of type `big.Int`.

Reply:
Oh nice, thanks for the tip! Updated